setup.py

```diff
@@ -61,7 +61,7 @@ def read_long_description():
         "requests",
         "six",
         "tabulate",
-        "sherlockml>=0.18.2",
+        "faculty>=0.18.2",
```

Comment:
The version constraint is now irrelevant since the new version of the `faculty` library is 0.19.0 (i.e., any version of `faculty` will satisfy it). I would remove it until there is a reason to set a minimum version.